src/components/common/Directors.vue

```diff
@@ -526,6 +526,22 @@ import { Actions, FilingStatus } from '@/enums'
 // Interfaces
 import { FormIF, BaseAddressIF, AlertMessageIF } from '@/interfaces'
 
+const emptyOfficer = {
+  firstName: '',
+  lastName: '',
+  middleInitial: ''
+}
+
+const emptyAddress = {
+  streetAddress: '',
+  streetAddressAdditional: '',
+  addressCity: '',
+  addressRegion: '',
+  postalCode: '',
+  addressCountry: '',
+  deliveryInstructions: ''
+}
```

Comment:
As this PR is against the most recent release branch, I only copied the minimum code here. Another commit in master fully implements this.